### PR TITLE
Change wait-list icon color and add a separate icon for admin registrations in the attendance list

### PR DESCRIPTION
**Pull request by kotlarz**
- Add the admin registration count to the attendance list header, as seen below.

- Add orange color to the registrations in the waiting list, and also add a seperate icon for admin registrations (with message if you hover over the icon), as seen below.

### CODE REVIEW

File: app/models.js [hunk 1]
```diff
@@ -102,6 +102,8 @@ export type EventRegistrationChargeStatus = 'manual' | 'succeeded' | 'failed';
 export type EventRegistration = {
   id: number,
   user: User,
+  adminRegistrationReason: string,
+  company: Company,
```
Comment (odinuge) [inline app/models.js]: company? Did I miss out on something?

Comment (kotlarz) [reply app/models.js]: I have honestly no idea... good eye 💯

File: app/routes/events/components/EventAdministrate/RegistrationTables.js [hunk 2]
```diff
@@ -61,13 +61,21 @@ export class RegisteredTable extends Component<Props> {
...
-        render: pool => (
+        render: (pool, registration) => (
           <TooltipIcon
-            content={pool ? 'Påmeldt' : 'Venteliste'}
+            content={
+              registration.adminRegistrationReason !== ''
+                ? 'Adminpåmeldt: ' + registration.adminRegistrationReason
+                : pool
+                ? 'Påmeldt'
+                : 'Venteliste'
```
Comment (orhanhenrik) [inline app/routes/events/components/EventAdministrate/RegistrationTables.js]: Nested ternary is not very readable 😞

Comment (kotlarz) [reply app/routes/events/components/EventAdministrate/RegistrationTables.js]: True, but couldn't be bothered as the linter wasn't complaining. #todo? 

We should add `no-nested-ternary` to our eslint config if we don't want that in the future :shrug:

**Pull request merged**